code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -46,6 +46,7 @@
 		var/mob/living/silicon/ai/A = owner.current
 		A.set_zeroth_law("")
 		A.remove_verb(/mob/living/silicon/ai/proc/choose_modules)
+		A.create_access_card(get_all_syndicate_access())
```

Comment:
I believe this is at the wrong location since this proc removes the traitor status and not adds it.
`finalize_traitor()` or `add_law_zero()` would be a better place for this